network-api/networkapi/buyersguide/templates/buyersguide/catalog.html

```diff
@@ -7,18 +7,17 @@
 {% block main_content_class %}{% endblock %}
 
 {% block hero %}
-<header class="container-fluid text-center page-header bg-gray" role="banner">
+<header
+  class="container-fluid text-center page-header bg-gray"
+  {% if page.get_banner != None %}style="background-image: url({% image page.get_banner max-1920x1280 as img %}{{ img.url }})"{% endif %}
```

Comment:
this only sets the actual image, and inherits the position/size/repeat from the #page-header CSS rule

Reply:
I was deciding if we should double up `1920x1280` so image renders better on retina screen. But since we mostly use pattern looking background on PNI, I think `1920x1280` should do good.